cegs_portal/search/view_models/v1/dna_features.py

```diff
@@ -282,3 +282,18 @@ def target_reo_search(cls, source_id: str):
         )
 
         return reg_effects
+
+    @classmethod
+    def non_targeting_reo_search(cls, genes: list[DNAFeature]):
+        source_features = DNAFeature.objects.filter(closest_gene_id=genes)
+        reg_effects = (
+            RegulatoryEffectObservation.objects.filter(
+                sources__in=Subquery(source_features.values("id")), targets=None
+            )
+        )
+        return reg_effects
+
```

Comment:
Please remove all these extra spaces. Files should just have one blank line at the end.